docs/landing/operations/deployment/README.md

```diff
@@ -0,0 +1,82 @@
+# Deployment
+
+This section handles the process of bringing your application from source code to any server.
+
+## Guides
+
+Guides for specific hosting platforms
+
+ * Deploy on [Heroku](./heroku/)
+
+## Build process
+
+In order to build your custom deployment, it is more than helpful to understand the build process. The following diagram illustrates the process.
+
+![Build process](./../../../assets/build-process.png)
+
+### Build trigger
+
+Depending on your CI setup you will usually have a trigger that initiates a build, e.g. pushing to a certain branch. You can obviously also trigger the build automatically.
+
+### Initialize the project
+
+```
+$ shopware-pwa init
+    --ci
+    --username [shopware-user]
+    --password [shopware-password]
+```
+
+Let's take this one apart. We set the `--ci` parameter to deactivate the interactive mode where the CLI would ask us for the Shopware credentials. Instead, we just provide them using the `--username` and `--password` credentials. These credentials are used to request the currently installed plugins in your Shopware instance.
+
+During CI, you're obviously not creating a new Nuxt project as the diagram above states. Instead, the CLI skips the initial setup, when there is a `nuxt.config.js` file in your project root.
+
+All artifacts of that step are placed in the `.shopware-pwa` directory.
+
+:::details 💡 Requests to Shopware during build process
+If you want to restrict requests by the build server for your API, make sure to exclude these routes:
+
+1. Obtain Admin API authentication token `/api/oauth/token`
+2. Fetch plugin assets `/api/v1/_action/pwa/dump-bundles`
+
+Step 2. will return two file links, one will be a .zip file containing all the resources from your plugins, the other one will be a .json file containing your plugins configuration (as made in the admin interface). Make sure, that these files are accessible by your build server.
+:::
+
+### Build the project
+
+```
+$ nuxt build
```

Comment:
it gives us potentially more possibilities to advice yarn build as we can update the script with optional params
```suggestion
$ yarn build
```